subqueries/subqueries.md

```diff
@@ -0,0 +1,276 @@
+1)
+
+A subquery is, essentially, a query within a query in SQL. A regular SQL query becomes a query when it is inserted into the `SELECT`, `FROM`, or `WHERE` clauses of a query. The subquery allows it's returned data to be used by the query it's found inside of.
+
+2)
+
+A subquery within a `SELECT` statement can be used when the results of a query should not be changed or influenced by the results of the subquery.
+
+3)
```

Comment:
this doesn't seem like the answer to the question?
"When would you employ a subquery?"

Reply:
true. Changed this.

Reply:
done